src/System.Transactions/tests/System.Transactions.Tests.csproj

```diff
@@ -15,6 +15,10 @@
     <Compile Include="TransactionScopeTest.cs" />
     <Compile Include="AsyncTest.cs" />
     <Compile Include="EnlistTest.cs" />
+    <Compile Include="CloneTxTests.cs" />
+    <Compile Include="LTMEnlistmentTests.cs" />
+    <Compile Include="HelperFunctions.cs" />
+    <Compile Include="TestEnlistments.cs" />
```

Comment:
Hmm, I've not run into that before so I'm not sure how to solve it.  Seems like a worthwhile thing to try.